Classes/Controller/AbstractController.php

```diff
@@ -24,6 +25,8 @@
 	private static $context;
 	private $contextBE;
 
+	public $settingsOrg;
```

Comment:
Don't use public class properties! It's not used at all, so I think you should remove it